src/core/agent/services/connectionService.ts

```diff
@@ -29,7 +29,6 @@ import {
 } from "../event.types";
 import {
   ConnectionHistoryItem,
-  KeriaContact,
```

Comment:
This should be fully removed now, I think it can back in with the merge. Replace all usage of `KeriaContact` with `Contact` from signify